portfolio/src/main/webapp/style.css

```diff
@@ -7,14 +7,16 @@
 #greeting-container {
   margin-top: 20px;
 }
-.paper{
-  display:grid;
+
+.paper {
+  display: grid;
   grid-template-rows: 50px 50px 50px;
-  grid-template-columns:50px 50px 50px;
-  gap : 10px;
+  grid-template-columns: 50px 50px 50px;
+  gap: 10px;
 }
-.box{
+
+.box {
   background-color:grey;
```

Comment:
nit: there's a missing space on this line.

There is also an HTML/CSS style guide, you've probably seen it, just putting it here in case :) https://google.github.io/styleguide/htmlcssguide.html